### PR TITLE
landingPage

### PR DESCRIPTION
**Pull request by ananyajain10**
landingPage

### CODE REVIEW

**Comment by Shorya7** [comment]
Same mistake again where are the files readme and license

File: Ananya_Jain/landingPage/app.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const nodemailer = require('nodemailer');
+const mongoose = require('mongoose');
+const ChannelModel = require('./models/channel')
+const dbUrl = "mongodb+srv://ananyajain:ananyaJAIN123@cluster0.j0todxb.mongodb.net/myDataBase?retryWrites=true&w=majority"
```
Comment (khareyash05) [inline Ananya_Jain/landingPage/app.js]:
```suggestion
const dbUrl = "mongodb+srv://ananyajain:<HIDE_THIS_URL>@cluster0.j0todxb.mongodb.net/myDataBase?retryWrites=true&w=majority"
```

File: Ananya_Jain/landingPage/app.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+// form
+app.get("/contact", function(req,res){
+  console.log("get");
+  res.sendFile("C:/Users/HP/Documents/Ananya/landing-page/landingPage/contact.html");
```
Comment (khareyash05) [inline Ananya_Jain/landingPage/app.js]: Not the way the file will work Ananya! Do you think we will be able to see this file?

File: Ananya_Jain/landingPage/app.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+  
+
+
+
+let transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'ananyajain866@gmail.com',
+    pass: 'ultwytbwkcgvbmmj'
```
Comment (khareyash05) [inline Ananya_Jain/landingPage/app.js]: Again Passwords!!!

**Review by khareyash05 — CHANGES_REQUESTED** [review]

File: Ananya_Jain/landingPage/app.js (hunk 1 above)
Comment (khareyash05) [reply Ananya_Jain/landingPage/app.js]: Add it in .env

**Comment by ananyajain10** [comment]
image


These readme and license files or  files missing inside LandingPage folder?

File: Ananya_Jain/landingPage/app.js (hunk 3 above)
Comment (ananyajain10) [reply Ananya_Jain/landingPage/app.js]: to hide them,  app.js should be inside .gitignore?

File: Ananya_Jain/landingPage/app.js (hunk 1 above)
Comment (ananyajain10) [reply Ananya_Jain/landingPage/app.js]: ok sir, got it.

File: Ananya_Jain/landingPage/app.js (hunk 2 above)
Comment (ananyajain10) [reply Ananya_Jain/landingPage/app.js]: I will correct this.

File: Ananya_Jain/landingPage/app.js (hunk 3 above)
Comment (khareyash05) [reply Ananya_Jain/landingPage/app.js]: No. if app.js will be in your gitignore we will not be able to see them. Do you know about dotenv package?

**Comment by Shorya7** [comment]
> image
> These readme and license files or files missing inside LandingPage folder?

Yes keep that as it is just add your folder into it, don't touch any other file or folder

**Comment by khareyash05** [comment]
> image
> These readme and license files or files missing inside LandingPage folder?

That looks good Leave it. Just do the above mentioned and you will be done.Great work

File: Ananya_Jain/landingPage/app.js (hunk 3 above)
Comment (ananyajain10) [reply Ananya_Jain/landingPage/app.js]: yes sir,
I will add <hidden passward> in public files.
and then add passwords in .eve file.
Is this correct?

Comment (khareyash05) [reply Ananya_Jain/landingPage/app.js]: create a .env file and import your env secrets using dotenv thats it

**Review by Shorya7 — CHANGES_REQUESTED** [review]
correct your folder name

**Review by khareyash05 — APPROVED** [review]

**Comment by ananyajain10** [comment]
Sir please review and merge this PR.

**Pull request merged**